package seedu.address.model.person;

import static org.junit.jupiter.api.Assertions.assertFalse;
import static org.junit.jupiter.api.Assertions.assertTrue;

import org.junit.jupiter.api.Test;

public class RemarkTest {

    @Test
    public void equals() {
        Remark remark = new Remark("Hello");

        assertTrue(remark.equals(remark));
        assertTrue(remark.equals(new Remark(remark.value)));
        assertFalse(remark.equals(1));
        assertFalse(remark.equals(null));
        assertFalse(remark.equals(new Remark("Bye")));
    }
}
